ner.py

```diff
@@ -8,11 +8,20 @@
 
 
 def load_flair_fr() -> SequenceTagger:
-    return SequenceTagger.load(path.join(Path.cwd(), 'models/fr-ner-wikiner-0.4.pt'))
+    # if loaded from downloaded models
+    # return SequenceTagger.load(path.join(Path.cwd(), 'models/fr-ner-wikiner-0.4.pt'))
+    return SequenceTagger.load("flair/ner-french")
```

Comment:
This will download the model in `~/.flair/` and doesn't use the already download model but if we change the model_path as it's used by huggingface library as model_id it won't work if the model isn't downloaded.

This would required more investigations.

Reply:
I remember that this path is given with the flair update dependency and it is downloaded already with the requirements but i have maybe wrong.